app/models/manageiq/providers/google/cloud_manager/provision_workflow.rb

```diff
@@ -5,6 +5,14 @@ def allowed_instance_types(_options = {})
     ems.each_with_object({}) { |f, h| h[f.id] = display_name_for_name_description(f) }
   end
 
+  def allowed_cloud_networks(_options = {})
+    return {} unless (src_obj = provider_or_tenant_object)
+
+    src_obj.all_cloud_networks.each_with_object({}) do |cn, hash|
```

Comment:
Per @Ladas' [comment](https://github.com/ManageIQ/manageiq/pull/16824#discussion_r161679886) This makes sense to me. Though adding a test per his comment would help prove the point.